Makefile

```diff
@@ -7,7 +7,7 @@ BIN := $(OUT)/rv32emu
 CONFIG_FILE := $(OUT)/.config
 -include $(CONFIG_FILE)
 
-CFLAGS = -std=gnu99 -O2 -Wall -Wextra
```

Comment:
Since we are modifying this, do we also want to add `-Wshadow` to catch more problems :) ?

Reply:
I think this depends on whether our coding style aims to forbid local variables from shadowing other variables?